addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -2181,6 +2182,64 @@ QUnit.test('chat window: chat header should not be clickable when thread has mul
     );
 });
 
+QUnit.test('2 chat windows: check shift operations have correct title with a LTR locale', async function (assert) {
+    assert.expect(2);
+
+    this.data['mail.channel'].records.push(
+        { is_minimized: true },
+        { is_minimized: true },
+    );
+    await this.start();
+
+    assert.strictEqual(
+        document.querySelector('.o_ChatWindowHeader_commandShiftPrev').title,
+        "Shift left",
+        "shift previous operation should be have 'Shift left' as title because locale is ltr"
+    );
+    assert.strictEqual(
+        document.querySelector('.o_ChatWindowHeader_commandShiftNext').title,
+        "Shift right",
+        "shift next operation should have 'Shift right' as title because locale is ltr"
+    );
+});
+
+QUnit.test('2 chat windows: check shift operations have correct title with a RTL locale', async function (assert) {
```

Comment:
Same remark as with the LTR test above, but with RTL:
Test is about textual representation of "previous"/"next" being mapped appropriately to "right"/"left" in RTL locale.